src/conf_mode/snmp.py

```diff
@@ -581,7 +581,11 @@ def apply(snmp):
     call('systemctl restart snmpd.service')
 
     # Enable AgentX in FRR
-    call('vtysh -c "configure terminal" -c "agentx" >/dev/null')
+    # This should be done for each daemon individually because common command
+    # works only if all the daemons started with SNMP support
+    frr_daemons_list = ['bgpd', 'ospf6d', 'ospfd', 'ripd', 'zebra']
```

Comment:
What is the benefit of enabling this only for selected daemons instead of all?

Reply:
If it should be enabled for a single daemon, we should read in the system rrr settings?

Reply:
There is one nuance with `vtysh` in this case: if not all daemons have the snmp module loaded (what is possible now) the command will not be applied to any of them. That is why we need to execute the command for each daemon individually.
This is some kind of workaround to keep SNMP settings logic simple and not involve using newly added CLI options there, which still can be the case after we will see that everything works as expected.